src/api/driftApi.ts

```diff
@@ -0,0 +1,77 @@
+import apiClient, { ApiClientResult } from './apiClient';
+
+interface journalpostOk {
+    sakId: string;
+    journalpostId: string;
+}
+export interface JournalpostSøknadOk extends journalpostOk {
+    søknadId: string;
+}
+export interface JournalpostIverksettOk extends journalpostOk {
+    behandlingId: string;
+}
+interface journalpostFeil {
+    sakId: string;
+    grunn: string;
+}
+export interface JournalpostSøknadFeil extends journalpostFeil {
+    søknadId: string;
+}
+export interface JournalpostIverksettFeil extends journalpostFeil {
+    behandlingId: string;
+}
```

Comment:
Bruk gjerne CamelCase på interface-navn.

```suggestion
interface JournalpostOk {
    sakId: string;
    journalpostId: string;
}
export interface JournalpostSøknadOk extends JournalpostOk {
    søknadId: string;
}
export interface JournalpostIverksettOk extends JournalpostOk {
    behandlingId: string;
}
interface JournalpostFeil {
    sakId: string;
    grunn: string;
}
export interface JournalpostSøknadFeil extends JournalpostFeil {
    søknadId: string;
}
export interface JournalpostIverksettFeil extends JournalpostFeil {
    behandlingId: string;
}
```